src/kili/presentation/client/project.py

```diff
@@ -265,6 +268,7 @@ def projects(
                 created_at_lte=None,
                 organization_id=organization_id,
                 tag_ids=tag_ids,
+                deleted=deleted,
```

Comment:
Add a quick description in the documentation above to explain what this field is doing

Reply:
Done ! ✅